### ISSUE TITLE
`block-scoped-var` reports the duplicate warnings of `no-undef`.

### ISSUE THREAD

**Issue by mysticatea** [eslint/eslint#3201]
`block-scoped-var` reports the duplicate warnings of `no-undef`.
I thought the warnings should be removed.

## test.js

```js
/* eslint block-scoped-var: 2, no-undef: 2 */
var a = b;
```

## Command & Actual

```
> eslint -v
v1.0.0-rc-3
> eslint test.js --no-eslintrc

test.js
  2:9  error  "b" used outside of binding context  block-scoped-var
  2:9  error  "b" is not defined                   no-undef

✖ 2 problems (2 errors, 0 warnings)
```

## Expected

```
test.js
  2:9  error  "b" is not defined  no-undef

✖ 1 problems (1 errors, 0 warnings)
```

----

In current implementation, it's just to remove a function.
See Also: https://github.com/eslint/eslint/pull/3126#issuecomment-125307709
